client/src/reducers/index.js

```diff
@@ -9,6 +9,19 @@ export default (state, action) => {
     case "FETCH_USER_SUCCESS": {
       return { ...state, user: action.user, loading: false };
     }
+    case "ADD_ITEM_TO_CART": {
+      // TODO handle duplicates?
+      return { ...state, loading: true, cart: [action.item] };
+    }
+    case "CREATE_PAYMENT_INTENT": {
+      return { ...state, loading: false };
+    }
+    case "CREATE_PAYMENT_INTENT_ERROR": {
+      return { ...state, loading: false };
+    }
+    case "FINISH_LOAD": {
+      return { ...state, loading: false };
+    }
```

Comment:
It seems there would be similar cases in the reducer. eg CREATE_PAYMENT_INTENT and FINISH_LOAD. Would it be better to reuse reducers?

Reply:
The `CREATE_PAYMENT_INTENT` and `CREATE_PAYMENT_INTENT_ERROR` are stubs. I'm working on the BE endpoints on my next PR, so I'm not entirely sure what the data will look like yet.